DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/ViewDelegateController.cs

```diff
@@ -79,18 +81,25 @@ public IActionResult Index(int delegateId)
                 model.DelegateInfo.Email = null;
 
             var baseUrl = config.GetAppRootPath();
-            if ((model.DelegateInfo?.IsActive ?? false) && (model.DelegateInfo.RegistrationConfirmationHash != null))
+            IClockUtility clockUtility = new ClockUtility();
+            if ((model.DelegateInfo?.IsActive ?? false) && (model.DelegateInfo.RegistrationConfirmationHash != null
+                && model.DelegateInfo.RegistrationConfirmationHashCreationDateTime?.AddDays(2) > clockUtility.UtcNow)

```

Comment:
The validity period will be moved to appsettings in https://hee-tis.atlassian.net/browse/TD-1863 please connect with @rshrirohit to make sure this is picked up here.

Reply:
@kevwhitt-hee This is now not required anymore I believe because we have reverted back the changes.